packages/typo3-guides-cli/src/Command/InitCommand.php

```diff
@@ -0,0 +1,247 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\GuidesCli\Command;
+
+use T3Docs\GuidesCli\Generation\DocumentationGenerator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use T3Docs\VersionHandling\Packagist\ComposerPackage;
+use T3Docs\VersionHandling\Packagist\PackagistService;
+
+/**
+ * You can run this command, for example like
+ *
+ * ddev exec packages/typo3-guides-cli/bin/typo3-guides init --working-dir=packages/typo3-guides-cli
+ *
+ */
+final class InitCommand extends Command
+{
+    protected static $defaultName = 'init';
+
+    protected function configure(): void
+    {
+        $this->setDescription('Initialize a new documentation project');
+        $this->addOption('working-dir', 'w', InputOption::VALUE_REQUIRED, 'If specified, use the given directory as working directory.');
+        $this->setHelp(
+            <<<'HELP'
+                This interactive command will help you to setup your project documentation.
+                To do so, it will ask you a few questions about your project and then
+                create a new documentation project in the working directory (default: current directory).
+
+                For more information, see:
+                https://docs.typo3.org/permalink/h2document:basic-principles
+                HELP
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        if ($input->getOption('working-dir')) {
+            $workdir = $input->getOption('working-dir');
+            assert(is_string($workdir));
```

Comment:
This is just a test against datatype "string", it will not cast value to string, right. So, if `zend.assertions` is -1 for production usage and the value is not string, it will not be casted.